fe/fe-core/src/main/java/org/apache/doris/cloud/datasource/CloudInternalCatalog.java

```diff
@@ -415,6 +416,18 @@ protected void afterCreatePartitions(long dbId, long tableId, List<Long> partiti
         }
     }
 
+    public void checkCreatePartitions(long dbId, long tableId, List<Long> partitionIds, List<Long> indexIds)
```

Comment:
This function seems not necessary, the implementation can be done in `CloudInternalCatalog.afterCreatePartitions()`